src/components/templates/Jazzbar/JazzBar/JazzBar.module.scss

```diff
@@ -0,0 +1,13 @@
+@use "scss/attendee/variables.scss";
```

Comment:
Should probably delete `src/components/templates/Jazzbar/JazzBar/JazzBar.scss`, not just leave empty